src/cpp/subscriber/StatisticsParticipantListener.cpp

```diff
@@ -128,7 +132,9 @@ std::string get_address(
 
 void StatisticsParticipantListener::on_participant_discovery(
         DomainParticipant* /*participant*/,
-        ParticipantDiscoveryInfo&& info)
+        ParticipantDiscoveryStatus reason,
+        const ParticipantBuiltinTopicData& info,
+        bool&)
```

Comment:
If variable not used, provide name and comment it, as done with the participant argument above.

```suggestion
        bool& /*what_was_this_argument_made_for*/)
```